templates/github-actions/scan-pull-request/frogbot-scan-pr-dotnet.yml

```diff
@@ -67,4 +67,8 @@ jobs:
 
           # [Optional, default: "FALSE"]
           # Displays all existing vulnerabilities, including the ones that were added by the pull request.
-          # JF_INCLUDE_ALL_VULNERABILITIES: "TRUE"
\ No newline at end of file
+          # JF_INCLUDE_ALL_VULNERABILITIES: "TRUE"
+
+          # [Optional, default: "TRUE"]
+          # Fails the Frogbot task if any security issue was found.
```

Comment:
...security issue was found. --> ...security issue is found.

Let's apply this change in all templates and in the README as well.